docs/run-tests-on-local.md

```diff
@@ -19,13 +19,13 @@ export WORK_DIR=${HOME}/edgex-taf
 # Arguments for run-tests.sh
 ${ARCH}: x86_64 | arm64
 ${SECURITY_SERVICE_NEEDED}: false | true
-${TEST_STRATEGY}: 1 (functional)
-${TEST_SERVICE}: all (default) | device-virtual | device-modbus | ${directory} under TAF/testScenarios/functionalTest/V2-API 
+${TEST_STRATEGY}: functional-test | integration-test mqtt | integration-test redis
+${TEST_SERVICE}:  v2-api (default) | device-virtual | device-modbus | ${directory} under TAF/testScenarios/functionalTest/V2-API 
```

Comment:
```suggestion
${TEST_STRATEGY}: functional-test | integration-test
${TEST_SERVICE}:  v2-api (default) | device-virtual | device-modbus | ${directory} under TAF/testScenarios/functionalTest/V2-API  | mqtt (integration-test) | redis (integration-test)
```

Reply:
revised

Reply:
Add two examples for run-tests-on-local.md(integration-test and functional-test), please check if it's appropriate. Thanks😊